projects/exui-common-lib/src/lib/components/generic-filter/generic-filter.component.html

```diff
@@ -1,40 +1,74 @@
-<div class="govuk-form-group">
-  <div class="xui-generic-filter" *ngFor="let field of config.fields">
-    <div class="govuk-form-group" [ngClass]="{'form-group-error': field.displayMinSelectedError || field.displayMaxSelectedError}">
-      <h3 *ngIf="field.title" class="govuk-heading-s">{{field.title}}</h3>
-      <p class="govuk-body" *ngIf="field.subTitle">{{field.subTitle}}</p>
-      <div class="govuk-body" [ngSwitch]="field.type">
-        <ng-container *ngSwitchCase="'checkbox'">
-          <div class="govuk-checkboxes govuk-checkboxes--small">
-            <div *ngFor="let item of field.options" class="govuk-checkboxes__item">
-              <input type="checkbox" [name]="'checkbox_' + item.key" [id]="'checkbox_' + item.key" [value]="item.key" [checked]="isSelected(field.name, item.key)" (change)="onSelectionChange($event)" class="govuk-checkboxes__input" [attr.field]="field.name"/>
-              <label [for]="'checkbox_' + item.key" class="govuk-label govuk-checkboxes__label" [attr.field]="field.name">{{item.label}}</label>
+<form [formGroup]="form" (ngSubmit)="applyFilter(form)">
+    <div class="contain-classes" *ngFor="let field of config.fields">
+      <hr *ngIf="field.lineBreakBefore" class="govuk-section-break govuk-section-break--visible elevated-break">
+      <div class="govuk-form-group xui-generic-filter"
+           [hidden]="hidden(field, form)"
+           [ngClass]="{'form-group-error': submitted && (form.get(field.name).errors?.minLength || form.get(field.name).errors?.maxLength)}">
+        <h3 *ngIf="field.title" class="govuk-heading-s">{{field.title}}</h3>
+        <p class="govuk-body" *ngIf="field.subTitle">{{field.subTitle}}</p>
+        <div class="govuk-body" [ngSwitch]="field.type">
+          <ng-container *ngSwitchCase="'select'">
+            <select class="govuk-select" (change)="updateSpecificPerson(field, form)" [attr.disabled]="disabled(field, form)" [name]="'select_' + field.name" [id]="'select_' + field.name" [formControlName]="field.name">
+              <option disabled selected hidden value="">{{field.disabledText}}</option>
+              <ng-container *ngFor="let item of field.options">
```

Comment:
no need for the container here, can put the ngFor directly on the option element